nipype/pipeline/engine/tests/test_utils.py

```diff
@@ -18,6 +18,11 @@
 from ..utils import merge_dict, clean_working_directory, write_workflow_prov
 
 
+def test_traits_version():
+    #just a temprorary test, testing CI
+    import traits
+    assert traits.__version__ >= "4.6.0"
```

Comment:
Actually that was my temporary test, when wanted to be sure that Travis indeed uses traits4.6 and still gives the segfault. 
I should probably just remove this test...